src/main/java/org/web3j/protocol/parity/JsonRpc2_0Parity.java

```diff
@@ -9,207 +8,324 @@
-import org.web3j.protocol.core.JsonRpc2_0Web3j;
+import org.web3j.protocol.admin.JsonRpc2_0Personal;
+import org.web3j.protocol.admin.methods.response.NewAccountIdentifier;
+import org.web3j.protocol.admin.methods.response.PersonalSign;
 import org.web3j.protocol.core.Request;
 import org.web3j.protocol.core.methods.request.Transaction;
 import org.web3j.protocol.core.methods.response.EthSendTransaction;
-import org.web3j.protocol.core.methods.response.VoidResponse;
-import org.web3j.protocol.parity.methods.response.NewAccountIdentifier;
-import org.web3j.protocol.parity.methods.response.PersonalAccountsInfo;
-import org.web3j.protocol.parity.methods.response.PersonalEcRecover;
-import org.web3j.protocol.parity.methods.response.PersonalListAccounts;
-import org.web3j.protocol.parity.methods.response.PersonalRejectRequest;
-import org.web3j.protocol.parity.methods.response.PersonalRequestsToConfirm;
-import org.web3j.protocol.parity.methods.response.PersonalSign;
-import org.web3j.protocol.parity.methods.response.PersonalSignerEnabled;
-import org.web3j.protocol.parity.methods.response.PersonalUnlockAccount;
+import org.web3j.protocol.parity.methods.response.BooleanResponse;
+import org.web3j.protocol.parity.methods.response.ParityAddressesResponse;
+import org.web3j.protocol.parity.methods.response.ParityAllAccountsInfo;
+import org.web3j.protocol.parity.methods.response.ParityDefaultAddressResponse;
+import org.web3j.protocol.parity.methods.response.ParityDeriveAddress;
+import org.web3j.protocol.parity.methods.response.ParityExportAccount;
+import org.web3j.protocol.parity.methods.response.ParityListRecentDapps;
+import org.web3j.protocol.parity.methods.response.SignerRejectRequest;
+import org.web3j.protocol.parity.methods.response.SignerRequestsToConfirm;
 
 /**
  * JSON-RPC 2.0 factory implementation for Parity.
  */
-public class JsonRpc2_0Parity extends JsonRpc2_0Web3j implements Parity {
+public class JsonRpc2_0Parity extends JsonRpc2_0Personal implements Parity {
 
     public JsonRpc2_0Parity(Web3jService web3jService) {
         super(web3jService);
     }
+    
+    @Override
+    public Request<?, ParityAllAccountsInfo> parityAllAccountsInfo() {
+        return new Request<>(
+                "parity_allAccountsInfo",
+                Collections.<String>emptyList(),
+                ID,
+                web3jService,
+                ParityAllAccountsInfo.class);
+    }
+
+    @Override
+    public Request<?, BooleanResponse> parityChangePassword(
+            String accountId, String oldPass, String newPass) {
+        return new Request<>(
+                "parity_changePassword",
+                Arrays.asList(accountId, oldPass, newPass),
+                ID,
+                web3jService,
+                BooleanResponse.class);
+    }
+
+    @Override
+    public Request<?, ParityDeriveAddress> parityDeriveAddressHash(
+            String accountId, String password, Map<String, Object> hashType, boolean toSave) {
```

Comment:
You should create a class for hashType rather then using a map here. For example see https://github.com/web3j/web3j/blob/master/src/main/java/org/web3j/protocol/core/methods/request/Transaction.java